lib/music_library_view.rb

```diff
@@ -0,0 +1,63 @@
+class MusicLibraryView
+  require "colorize"
```

Comment:
#### Code Quality - Nitpick 
Prefer single-quoted strings when you don't need string interpolation or special symbols such as ```\t```, ```\n```, ```'```